src/Megamix/Patches.cpp

```diff
@@ -94,29 +82,82 @@ namespace Megamix::Patches {
         return cond | cmp_imm_base | reg | value;
     }
 
+    void AddExtraRowsToFront() {
+        std::vector<MuseumRow>      extraMuseumRows {};
+        std::vector<MuseumRowColor> extraMuseumRowColors {};
+
+        // add tickflows to extra rows
+        std::array<u16, 5> newRowIds { 0x101, 0x101, 0x101, 0x101, 0x101 };
+        size_t newRowLength = 0;
+
+        auto PushNewRows = [&]() {
+            extraMuseumRows.emplace_back(newRowIds, "", 0, 0);
+            extraMuseumRowColors.emplace_back(0x424242ff, 0x00000000);
+
+            newRowIds = { 0x101, 0x101, 0x101, 0x101, 0x101 };
+            newRowLength = 0;
+        };
+
+        for (auto &pair : config->tickflows) {
+            newRowIds[newRowLength] = pair.first;
+            newRowLength += 1;
+
+            if (newRowLength == 5) {
+                PushNewRows();
+            }
+        }
+
+        if (newRowLength != 0) {
+            PushNewRows();
```

Comment:
what happens if there's not enough games to fill a full row?

Reply:
the rows don't have to be filled, the constructor for the row guess the column count based on the first null id

Reply:
alright but if the count is 2, wouldn't that break?

Reply:
oh it would...